pilz_robot_programming/Readme.rst

```diff
@@ -145,14 +145,16 @@ Custom Reference Frame
 ^^^^^^^^^^^^^^^^^^^^^^
 .. code-block:: python
 
-    r.move(Ptp(goal=Pose(position=Point(0, 0, 0.1)), reference_frame="prbt_tcp"))
+    r.move(Ptp(goal=PoseStamped(header=Header(frame_id="prbt_tcp"),
+                                pose=Pose(position=Point(0, 0, 0.1)))))
 
 .. code-block:: python
 
     r.move(Ptp(goal=Pose(position=Point(0, -0.1, 0)), reference_frame="prbt_link_3", relative=True))
 
-All three move classes :py:class:`.Ptp`, :py:class:`.Lin` and :py:class:`.Circ` can be executed within a custom reference frame.
-In this case, the passed goal pose will be seen relative to this coordinate system instead of the default system: ``prbt_base``
+For all three move classes :py:class:`.Ptp`, :py:class:`.Lin` and :py:class:`.Circ` you can define a custom reference frame.
+Passing a PoseStamped with the Header set to any valid tf2 ``frame_id`` is supported besides an extra argument ``reference_frame``.
```

Comment:
Please note the constraint on rospy.Time()

Reply:
I've updated the docstring of `BaseCommand` with that information.

Why tell in the tutorial, what *not* to do? That's a Readme and not an API spec: [don't read this](https://www.psychologytoday.com/us/blog/management-rewired/201305/dont-read).